samples/smp_pi/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _smp_pi:
```

Comment:
You'll also need to add a line in ``samples/index.rst`` to reference this new folder of samples (``smp/*``)